### PR TITLE
fix: ignore dperf on unformatted/unavailable/unmounted drives

### PR DESCRIPTION
**Pull request by harshavardhana**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license] (https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
fix: ignore dperf on unformatted/unavailable/unmounted drives

## Motivation and Context
do not run `dperf` on drives that are showing as offline in `minio`

## How to test this PR?
Observe path a drive is unmounted, would lead to path being on
the root/boot disk - dperf inadvertently runs on the boot disk 
showing a weird skew in performance, instead we should it as a faulty
disk.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

**Review by kannappanr — APPROVED** [review]
LGTM

File: cmd/speedtest.go [hunk 1]
```diff
@@ -274,6 +280,12 @@ func driveSpeedTest(ctx context.Context, opts madmin.DriveSpeedTestOpts) madmin.
 				}
 				results = append(results, result)
 			}
+			for _, inp := range ignoredPaths {
+				results = append(results, madmin.DrivePerf{
+					Path:  inp,
+					Error: errFaultyDisk.Error(),
```
Comment (jiuker) [inline cmd/speedtest.go:286]: Do we need `errSkipPath` for this? This err is confusing.

**Review by jiuker — APPROVED** [review]
LGTM.

File: cmd/speedtest.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/speedtest.go:286]: No the error is correct @jiuker since we need to tell that drive is faulty so its ignored.

**Pull request merged**